_posts/2015-07-17-cassandra-write-performance-on-gce-and-aws.md

```diff
@@ -5,4 +5,91 @@ layout: post
 comments: true
 ---
 
-... or how we learned to create cross-cloud repeatable benchmarks for everybody. 
+*... or how we created cross-cloud tunable, repeatable benchmarks for everybody.*
+
+tl;dr - We achieved better Cassandra performance on GCE vs. Amazon, at close to half the cost.
+
+Benchmarking and performance are interesting problems, especially in today’s growing [microservices](https://en.wikipedia.org/wiki/Microservices) scene. It used to be a question of “how does this hardware compare to that hardware,” but as computing and service-oriented architectures grow the question has evolved. How does my [cloud and application](http://12factor.net/) stack handle this? It’s no longer enough to run your favorite machine benchmark tool on your web server and call it a day.
+
+Measuring every microservice in your stack, from backend to frontend, is a complex task. We started thinking about how you would model a system to benchmark all of these services. It’s not just a matter of measuring the performance of one service, but also its interactions with other services. Now multiply that by every config option for every service, like Cassandra, which has hundreds of options that can affect performance. And let’s not forget everything that you need to get Cassandra running, all those things have knobs too.
```

Comment:
These two paragraphs are straight from the about page, I'd nix most of this and just take what's important to the post